src/rust/bitbox02-rust/src/hww/api/cardano/sign_transaction/cbor.rs

```diff
@@ -0,0 +1,221 @@
+// Copyright 2021 Shift Crypto AG
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+use super::super::params;
+use super::super::pb;
+use super::super::Error;
+
+use alloc::vec::Vec;
+
+use blake2::digest::Update;
```

Comment:
doesn't this come from [digest crate](https://docs.rs/digest/0.9.0/digest/trait.Update.html) and should be `use digest::Update` here?

Reply:
Done